hydroflow/src/scheduled/mod.rs

```diff
@@ -2,44 +2,50 @@ pub mod collections;
 pub mod ctx;
 pub mod handoff;
 #[cfg(feature = "variadic_generics")]
-pub mod handoff_list;
 pub mod input;
 pub mod query;
+pub mod state;
 pub(crate) mod subgraph;
 pub mod util;
 
-use futures::stream::Stream;
+mod handoff_list;
+pub use handoff_list::HandoffList;
+
+use std::any::Any;
 use std::cell::Cell;
 use std::cell::RefCell;
 use std::collections::VecDeque;
 use std::marker::PhantomData;
 use std::pin::Pin;
 use std::rc::Rc;
 use std::sync::mpsc::{self, Receiver, RecvError, SyncSender, TrySendError};
-use std::task::{Context, Poll};
+use std::task::{self, Poll};
 
+use futures::stream::Stream;
 use ref_cast::RefCast;
 
 use crate::tl;
 use ctx::{InputPort, OutputPort, RecvCtx, SendCtx};
 use handoff::{Handoff, HandoffMeta, VecHandoff};
+use state::StateHandle;
 #[cfg(feature = "variadic_generics")]
```

Comment:
does this cfg still apply?

Reply:
Yeah, although that cfg is pretty much required at the current moment for hydroflow to be useful